README.md

```diff
@@ -68,6 +68,7 @@ If the filename ends with `.url` suffix, the content will be processed as a URL
 | `REQ_RETRY_CONNECT`        | How many connection-related errors to retry on for any http request (`*.url` triggered requests, requests to `REQ_URI` and k8s api requests)                                                                                                                                                                                        | false    | `10`                                      | integer |
 | `REQ_RETRY_READ`           | How many times to retry on read errors for any http request (`.url` triggered requests, requests to `REQ_URI` and k8s api requests)                                                                                                                                                                                                 | false    | `5`                                       | integer |
 | `REQ_RETRY_BACKOFF_FACTOR` | A backoff factor to apply between attempts after the second try for any http request (`.url` triggered requests, requests to `REQ_URI` and k8s api requests)                                                                                                                                                                        | false    | `1.1`                                     | float   |
+| `REQ_IGNORE_INITIAL_EVENT` | Set to `true` to ignore requests for first events or initial list for configmaps and secrets events. Applicable only when `IGNORE_ALREADY_PROCESSED` is enabled.                                                                                                                                                                    | false    | `false`                                   | boolean |
```

Comment:
I think we should explain better under which circumstances a request will be skipped (accounting in the batch logic).

Reply:
Improved - please take a look now 🙂